README.md

```diff
@@ -21,6 +21,72 @@ Import the components you need, example:
 import { Button, Popover, Tooltip } from 'reactstrap';
 ```
 
+## Getting Started
+
+You don’t have to use [Reactstrap](https://reactstrap.github.io) together with React but it is a popular library for integrating Bootstrap with React apps. If you need it, you can integrate it with Create React App by following these steps:
+
+Install Reactstrap and Bootstrap from NPM. Reactstrap does not include Bootstrap CSS so this needs to be installed as well:
+
+```
+npm install react-bootstrap --save
+npm install bootstrap@4 --save
+```
+
+Import Bootstrap CSS and optionally Bootstrap theme CSS in the ```src/index.js``` file:
+
+```js
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/css/bootstrap-theme.css';
+```
+
+Import required Reactstrap components within ```src/App.js``` file or your custom component files:
+
+```js
+import { Navbar, Jumbotron, Button } from 'reactstrap';
+```
+
+Now you are ready to use the imported Reactstrap components within your component hierarchy defined in the render method. Here is an example `App.js` redone using Reactstrap:
+
+```js
+import React, { Component } from 'react';
+import { Grid, Navbar, Jumbotron, Button } from 'reactstrap';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Navbar inverse fixedTop>
+          <Grid>
+            <Navbar.Header>
```

Comment:
The gist content will need to be updated to use the component API according to the reactstrap docs. See https://reactstrap.github.io/components/navbar/

Same goes for the button example.